deploy/clowdapp.yaml

```diff
@@ -355,7 +355,7 @@ parameters:
 - name: IMAGE_TAG
   required: true
 - name: IMAGE
-  value: quay.io/cloudservices/playbook-dispatcher
+  value: quay.io/redhat-services-prod/hcc-integrations-tenant/playbook-dispatcher
```

Comment:
This change will only affect ephemeral. Stage and prod already pull the Konflux image because of the `IMAGE` param in app-interface.